plugin/src/main/java/com/denizenscript/denizen/scripts/commands/item/MapCommand.java

```diff
@@ -32,7 +32,7 @@ public MapCommand() {
 
     // <--[command]
     // @Name Map
-    // @Syntax map [<#>/new:<world>] [reset:<location>/image:<file> (resize)/script:<script>] (x:<#>) (y:<#>)
+    // @Syntax map [<#>/new:<world>] [reset:<location>/image:<file> (resize)/script:<script>] (scale:<value>) (tracking:<true/false>) (x:<#>) (y:<#>)

```

Comment:
This should be `(tracking)`, rather than a boolean input (what's the difference between `tracking:false` and just leaving off the argument?)

Reply:
`tracking:false` specifically turns off the tracking if the map is within the map's displayed area.
i.e. the player's location on the map, when they're holding it.

Reply:
That doesn't answer the question.

The correct answer to the question, for reference, is "Literally nothing".